python/ppml/src/bigdl/ppml/fl/nn/pytorch/estimator.py

```diff
@@ -36,7 +35,13 @@ def __init__(self, model: nn.Module, loss_fn, optimizer: torch.optim.Optimizer,
         self.fl_client = FLClient(target)
         self.loss_history = []
 
-    
+    @staticmethod
+    def from_torch(model: nn.Module,
+                   loss_fn,
+                   optimizer: torch.optim.Optimizer,
+                   target="localhost:8980"):
+        return PytorchEstimator(model, loss_fn, optimizer, target=target)
+
     
     def add_server_model(self, model: nn.Module, loss_fn=None, optimizer_cls=None, optimizer_args={}):
```

Comment:
instead of `add_server_model`, make it 
```python
Estimator.from_torch(client_model, 
                     client_optimizer, 
                     server_model=None, #only set when client_id==1
                     server_optimizer=None, 
                     ...)
```

Reply:
I don't think we need `add_server_model` here

Reply:
This is just a wrapper method and with `fl.Estimator` I guess this `pytorch.estimator` could be no longer exposed to users?

Reply:
The `add_server_model` is redundant and can be removed.

Reply:
Currently the style is `fl.Estimator.from_torch` creates an instance of `PytorchEstimator` and pass `server_model` to the args of `add_server_model` so this method is not redundant I guess?

Reply:
then make it internal

Reply:
`fl.Estimator` and `pytorch.Estimator` has no inheriting relation so private/protected method could not be called across class. 

Current API is `fl.Estimator.from_torch` and `fl.Estimator.from_keras`, should we change to `pytorch.Estimator` and `tensorflow.Estimator` respectively?